source/wp-content/themes/wporg-developer-2023/patterns/front-page-content.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * Title: Front Page Content
+ * Slug: wporg-developer-2023/front-page-content
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"16px","right":"var:preset|spacing|edge-space","bottom":"16px","left":"var:preset|spacing|edge-space"}},"elements":{"link":{"color":{"text":"var:preset|color|white"}}}},"backgroundColor":"charcoal-2","textColor":"white","className":"is-sticky wporg-front-page-breadcrumb","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull is-sticky wporg-front-page-breadcrumb has-white-color has-charcoal-2-background-color has-text-color has-background has-link-color" style="padding-top:16px;padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:16px;padding-left:var(--wp--preset--spacing--edge-space)"><!-- wp:group {"align":"full","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignfull"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:heading {"className":"is-style-default","fontSize":"small","fontFamily":"inter"} -->
+<h2 class="wp-block-heading is-style-default has-inter-font-family has-small-font-size"><?php esc_attr_e( 'Developers', 'wporg' ); ?></h2>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:navigation {"textColor":"white","backgroundColor":"charcoal-2","className":"is-style-dropdown-on-mobile","style":{"spacing":{"blockGap":"24px"}},"fontSize":"small"} -->
```

Comment:
I kept this _second_ navigation around for 2 reasons:
- Making the colors match is possible with CSS, but all the styles have `important` and we lose the ability to easily customize it in the editor.
- The site-breadcrumb doesn't match the designs. It would require extra CSS to make that work. 

Having 2 link lists is not super dry, but the best of 2 evils in this case.

Reply:
We could try `__unstableLocation` to reuse menu items in different navigation blocks by the location name, but it's `unstable` and requires a little setup in the local environment to make it work. I keep meaning to do an experiment branch to wporg-parent when we're between projects but…  😩